css/ampstart-base/elements/initialcap.css

```diff
@@ -23,5 +23,8 @@
   color: var(--color-font-secondary);
   font-size: var(--h1);
   font-weight: var(--bold-font-weight);
-  margin-left: -2px;
```

Comment:
This is intentional - you need to be looking at dropcap for the effect you want !